src/components/Chart/TimeBarsAnnotations.js

```diff
@@ -16,6 +16,19 @@ const labelGauger = createTextGauger(LABEL_FONT, {
   html: true
 })
 
+const convertTextAlignment = alignment => {
+  switch (alignment) {
+    case 'left':
+      return 'start'
+    case 'center':
+      return 'middle'
+    case 'right':
+      return 'end'
+    default:
+      return 'middle'
+  }
+}
+
```

Comment:
that's a matter on taste but I tend to prefer dictionaries for this (more concise):
```
const textAlignmentDict = {
  left: 'start',
  center: 'middle',
  right: 'end'
}
```

and then 
```
textAnchor = textAlignmentDict[annotation.textAlignment] || 'middle'
```